schema.txt

```diff
@@ -0,0 +1,27 @@
+Teams - represents a team
+-> Id
+-> [UserIds]
+-> Name
+-> Header
+-> Description
+-> ProjectEndDate - currently working on a project
+-> CanStartAtMin - minimum notice period of the team member
+-> CanStartAtMax - minimum notice period of the team member
+-> [TagIds]
+
+-Accounts - represents a teampot account
```

Comment:
@aye-dev I assume this is a user account. Don't you want to change the name to User?

Reply:
Yep, means user account.  UserAccounts then? We'll also need an entity for clients (ClientAcconts) like companies or recruitment agencies (set of user accounts) that for instance have some search configs and advanced features enabled.

Reply:
I would call it simply `user` as welll